test/pa11y.conf.js

```diff
@@ -0,0 +1,62 @@
+const config = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).